CONTRIBUTING.md

```diff
@@ -20,6 +20,36 @@ Use the [Discussions](https://github.com/github/gh-gei/discussions) tab in this
 
 All work done by the maintainers of this repo is tracked in this repo using Issues. We have a hierarchical backlog with Epics at the top, broken down into Batches then broken down to Tasks (epic/batch/task is indicated via labels on the issues). You can see an example Epic and navigate down from there [here](https://github.com/github/gh-gei/issues/101).
 
+
+## Debugging code 
+
+There are two ways to debug code within VS code. 
+
+### Run GH-GHEI command locally
+Run the following commands in your terminal depending on the provider you are looking to run the commands for. 
+
+Azure DevOps: 
+`dotnet run --project src/ado2gh/ado2gh.csproj -- <<INSERT COMMAND HERE>>`
```

Comment:
In most CLI documentation (i.e. man pages), straight brackets explicitly document optional parameters.

For example, here is `man ls`:

```
SYNOPSIS
       ls [OPTION]... [FILE]...
```

And here's `man top`:

```
SYNOPSIS
       top [options]
```

It would be more conformist to also use straight brackets here:

```suggestion
`dotnet run --project src/ado2gh/ado2gh.csproj -- [command]`
```

If you dig, let's also use this with the BBS and GitHub commands, too :+1: